infrastructure/stacks/application/scripts/database_populator/database_populator.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import pandas as pd
+from decimal import Decimal
+import boto3
+from io import BytesIO
+
+# Initialize the S3 and DynamoDB clients
+s3 = boto3.client("s3")
+dynamodb = boto3.resource("dynamodb")
+ddbclient = boto3.client("dynamodb")
+
+# Specify the S3 bucket name and Excel file key
+bucket_name = "nhse-uec-cm-dev-databucket"
```

Comment:
References here to cm

Reply:
Nachi is working on updating this script. When that's done will bring in the updated script with reference to 'sm'